### DISCUSSION TITLE
Setup guide for leptos_oidc and Rauthy

### DISCUSSION THREAD

**Discussion by erlend-sh** [sebadob/rauthy#140] [General]
@kerkmann made this: https://crates.io/crates/leptos_oidc

I mentioned Rauthy and so they followed up with this awesome setup guide ✨

https://gitlab.com/kerkmann/leptos_oidc/-/blob/add-rauthy-support/docs/backends/rauthy.md

Might wanna link to that in the readme or docs.

**Comment by kerkmann** [comment]
Thanks for mentioning it! I've merged the changes to the main: [https://gitlab.com/kerkmann/leptos_oidc](https://gitlab.com/kerkmann/leptos_oidc)

Feedback is always welcome, thanks! :heart:

**Comment by sebadob** [comment]
Oh that's awesome. I didn't know yet, that this crate exists. I am using leptos myself as well in other places and projects and it will be added to the [nioca-client](https://github.com/sebadob/nioca-client) in the future.

I actually plan on adding a section to the docs in the future where I describe the setup with different clients as well.
